tests/features/content-types.feature

```diff
@@ -4,27 +4,64 @@ Feature: Common fields across the content types
   As an editor
   I should have common fields configured the same for each content type.
 
-  Scenario Outline: Fields have the correct description.
-    Given I am logged in as a user with the "create oe_event content, create oe_news content, create oe_page content, create oe_policy content, create oe_project content, create oe_publication content, create oe_call_tenders content, access content" permission
+  Scenario Outline: Common fields have the correct description.
+    Given I am logged in as a user with the "create oe_event content, create oe_news content, create oe_page content, create oe_policy content, create oe_project content, create oe_publication content, create oe_call_tenders content, create oe_call_proposals content, create oe_organisation content, create oe_person content, create oe_consultation content, access content" permission
     When I visit "the <content_type> creation page"
-    Then I should see the text "The ideal length is 50 to 60 characters including spaces." in the "title form element"
     And I should see the text "If it must be longer, make sure you fill in a shorter version in the Alternative title field." in the "title form element"
```

Comment:
This should be moved to the "Title field have the correct description." scenario